Calibration/HcalAlCaRecoProducers/python/alcaisotrk_cfi.py

```diff
@@ -38,6 +38,5 @@
 # Prescale events only containing isolated tracks in the range
                          MomentumRangeLow  = cms.double(20.0),
                          MomentumRangeHigh = cms.double(40.0),
-                         PreScaleFactor    = cms.int32(2)
+                         PreScaleFactor    = cms.int32(10)
```

Comment:
Should this PR not change anything in term of rates? why the PreScaleFactor is different?

Reply:
We shall keep the prescale factor high for the HLT and the rate will be reduced for this by a factor of 2